libraries/utils/src/render-lines-on-sp.ts

```diff
@@ -1,31 +1,27 @@
 import * as DG from 'datagrok-api/dg';
 import { Subject } from 'rxjs';
-
-export enum LineDirection {
-    pointToLowerValue = 'lower',
-    pointToHigherValue = 'higher',
-}
-
-export type ILineOpts = {
-    points: Uint32Array;
-    id: number;
-    color?: string;
-    directionCol?: string
-    direction?: LineDirection;
-    opacity?: number;
-    width?: number;
+import BitArray from './bit-array';
+
+export type ILines = {
```

Comment:
I suggest to rename it to ILineSeries, this is typically what such things are called